components/Slider/src/MDCSlider.h

```diff
@@ -335,13 +335,21 @@ IB_DESIGNABLE
  */
 @property(nonatomic, strong, null_resettable) UIColor *trackBackgroundColor UI_APPEARANCE_SELECTOR;
 
-/** When @c YES, haptics are enabled. The haptics casue a light impact reaction when the slider
- reaches the minimum or maximum value.
+/** When @c YES, haptics for min and max are enabled. The haptics casue a light impact reaction when
+ the slider reaches the minimum or maximum value.
 
  Defaults to @c YES in iOS 10 or later, @c NO otherwise
  */
 @property(nonatomic, assign) BOOL hapticsEnabled;
 
+/** When @c YES, haptics for any value change are enabled for discrete sliders. The haptics casue
+ a light impact reaction when the slider value changes for discrete sliders. Can only be set to yes
+ for discrete sliders.
+
+ Defaults to @c NO
+ */
+@property(nonatomic, assign) BOOL fullHapticsEnabled;
```

Comment:
I feel that the naming "fullHapticsEnabled" may be ambiguous to some users. I wonder if we should be more explicit about what is being enabled when this property is set to YES. I would try to see how Apple names similar properties that create similar behaviors.

One suggestion would be to name the property `shouldEnableHapticsForAllDiscreteValues`. In Objective-C it's recommended for properties to be "self-documenting".